packages/web-client/src/webdav/moveFiles.ts

```diff
@@ -9,8 +14,15 @@ export const MoveFilesFactory = ({ sdk }: WebDavOptions) => {
       { path: sourcePath },
       targetSpace: SpaceResource,
       { path: targetPath },
-      options?: { overwrite?: boolean }
+      options?: { overwrite?: boolean; shareJailId?: string }
```

Comment:
@JanAckermann please remove the shareJailId option, not used but imported as discussed.